acceptance/Gemfile

```diff
@@ -1,7 +1,7 @@
 source "https://rubygems.org"
 
 gem "chef-acceptance", github: "chef/chef-acceptance"
-gem "test-kitchen", github: "test-kitchen/test-kitchen"
+gem "test-kitchen"
```

Comment:
I know we wanted to start using released version of Test Kitchen, but I don't know if we are okay with taking every new version.  Since chef acceptance is still moving pretty fast I'm inclined to leave this as-is (especially since it fixes Manhattan).